docs/restic_backup.md

```diff
@@ -0,0 +1,75 @@
+# Run LIME with Restic Docker Backup
+
+## Overview
+
+This documentation covers setting up automated backups for Docker Compose applications using Restic, a modern backup tool that provides fast, secure, and deduplicated backups. The setup uses the `mekomsolutions/restic-compose-backup` Docker image to automatically backup Docker containers and volumes to a local directory. See [mekomsolutions/restic-compose-backup](https://github.com/mekomsolutions/restic-compose-backup) for more details.
+
+
+## Installation and Setup
+
+### 1. Create Backup Directory
+
+Create a dedicated directory for your backups:
+
+```bash
+# Create backup directory
+sudo mkdir -p ~/backups/restic_data
+
+# Set appropriate permissions
+sudo chmod 700 ~/backups/restic_data
+
+# Change ownership if needed
+sudo chown $USER:$USER ~/backups/restic_data
+```
+
+### 2. Environment Configuration
+
+Create a `run/docker/concatenated.env` file with your backup configuration:
+
+```bash
+# Restic Backup Configuration
+BACKUP_PATH=~/backups/restic_data
+RESTIC_PASSWORD=your-strong-password-here
+RESTIC_KEEP_DAILY=7
+RESTIC_KEEP_WEEKLY=4
+RESTIC_KEEP_MONTHLY=6
+RESTIC_KEEP_YEARLY=2
+LOG_LEVEL=INFO
+CRON_SCHEDULE=0 0 * * *
```

Comment:
@pirupius should we add these to our environment files so that we do not need to manually add them on each deployment or we should first test them on DEV?